dts/bindings/sensor/st,ism330dhcx-common.yaml

```diff
@@ -10,3 +10,26 @@ properties:
         This pin defaults to active high when produced by the sensor.
         The property value should ensure the flags properly describe
         the signal that is presented to the driver.
+
+    accel-range:
+      type: int
+      required: false
+      default: 2
+      description: Range in g. Default is power-up configuration.
+      enum:
+        - 16 # 16g (0.488 mg/LSB)
```

Comment:
You might want to move these comments to the `description` block so they show up in the HTML. Your call; I'm not blocking.

Reply:
Seems a good idea.